README.md

```diff
@@ -58,17 +56,22 @@ const { EventHubClient, EventHubReceiver, EventPosition } = require('azure-event
 const client = EventHubClient.fromConnectionString('Endpoint=sb://my-servicebus-namespace.servicebus.windows.net/;SharedAccessKeyName=my-SA-name;SharedAccessKey=my-SA-key', 'myeventhub');
 
 function async main() {
-  const receiver = await client.createReceiver("1", { eventPosition: EventPosition.fromEnqueuedTime(Date.now()) });
-  // Super important - Please add an event listener for the error event.
-  receiver.on("error", (err) => {
-    console.log("An error occurred on the receiver ", receiver.name, err);
-  });
+  const receiver = client.createReceiver("1", { eventPosition: EventPosition.fromEnqueuedTime(Date.now()) });
   // For receiving messages:
-  // Option 1: Add an event listener to the "message" event.
-  receiver.on("message", (eventData) => {
+
+  // Option 1: Add handlers for message and error and provide them to the start(). <<<<<<<<<<<<<<<
```

Comment:
><<<<<<<<<<<<<<< [](start = 83, length = 15)

intended or merge artifact?

Reply:
intended to call out start() as an option for receiving messages.